spec/arrays.dd

```diff
@@ -525,6 +525,9 @@ assert(a == [9, 6, 1]);
         has zero elements in it.
         )
 
+        $(P If the element type defines matching overloaded operators,
+        those methods must be `pure nothrow @nogc`.)
```

Comment:
ATM you get a druntime error if e.g. `opBinary` violates those. `@system` seems to be fine.